src/components/apikey/APIKeyForm.tsx

```
@@ -0,0 +1,312 @@
+          isDisabled={!isCreate}
+        >
+          <FormSelectOption key="placeholder" value="" label="Select a namespace" isDisabled />
+          {namespaces.map((ns) => (
+            <FormSelectOption key={ns} value={ns} label={ns} />
+          ))}
+        </FormSelect>
+        <FormHelperText>
+          <HelperText>
+            <HelperTextItem>
+              Namespace where the APIKey will be created
+            </HelperTextItem>
```

ESLint

[error] Replace `⏎··············Namespace·where·the·APIKey·will·be·created⏎············` with `Namespace·where·the·APIKey·will·be·created`

(prettier/prettier)